### ISSUE TITLE
FPL - a full platform abstraction single file header public domain C++ library

### ISSUE THREAD

**Issue by f1nalspace** [r-lyeh/single_file_libs#107]
Can you please add my platform abstraction single file header public domain C++ library i have written recently?:

https://github.com/f1nalspace/final_game_tech/blob/master/final_platform_layer.hpp

Thanks!

**Comment by r-lyeh** [comment]
Closing ticket. Link(s) should be already in the list. Please double-check.

**Comment by f1nalspace** [comment]
Hi there,

all libraries are listed, but on uncategorized.

Can you please add them to its respective category?

final_platform_layer.h -> operating system features -> system (new category)
final_dynamic_opengl.h -> graphics (3d)
final_tiletrace.hpp -> graphics (2d)
final_memory.h -> operating system features -> system (new category)

final_xml.h -> parsing -> XML (i know you are not a fan of XML, but there are dozens other XML headers, so just put them together in its own category)

**Comment by r-lyeh** [comment]
Yup, I plan to merge all the sections together within next hours. "Uncategorized" section will be gone.
